server/meeting.go

```diff
@@ -16,7 +16,7 @@ var (
 type Meeting struct {
 	ChannelID     string         `json:"channelId"`
 	Schedule      []time.Weekday `json:"schedule"`
-	HashtagFormat string         `json:"hashtagFormat"` // Default: {ChannelName}-Jan02
+	HashtagFormat string         `json:"hashtagFormat"` // Default: {ChannelName}-Jan-2
```

Comment:
Why did the default change here?

Reply:
Hashtags with `-` were returning no results for me, after some digging found out:
![Screenshot_1](https://user-images.githubusercontent.com/85980820/135958490-23d1f75d-03f7-483a-8a4a-ec78e50b263f.png)
[ref](https://docs.mattermost.com/messaging/searching-in-mattermost.html)
2/5, our plugin should be database agnostic but not sure if I should include these changes in this PR.
@mickmister

Reply:
I'm not able to reproduce this issue using postgress using `#foo-02`. Is there anything more specific case that causes issues?